component/src/main/java/io/siddhi/extension/io/http/source/SSEServerSource.java

```diff
@@ -71,7 +71,7 @@
 /**
  * Http source to receive sse events.
  */
-@Extension(name = "sse", namespace = "source",
+@Extension(name = "sse-server", namespace = "source",
```

Comment:
@sajithshn is this correct ? in the review we discussed to keep it as it is right

Reply:
for, sink we decided to change since it acts as a server. And for the source we kept as it is.

Reply:
so this is source right ? isnt this should be just sse ?